server/rooms.ts

```diff
@@ -217,7 +219,7 @@ export abstract class BasicRoom {
 	 */
 	nthMessageHandlers: Map<MessageHandler, number>;
 
-	constructor(roomid: RoomID, title?: string, options: Partial<RoomSettings> = {}) {
+	constructor(roomid: RoomID, title?: string, section: RoomSection = 'none', options: Partial<RoomSettings> = {}) {
```

Comment:
Why is `section` its own param instead of just being a roomsetting (and defaulting to 'none')?

Reply:
It's so admins can specify sections upon room creation if they want